src/main/scala/akka/persistence/cassandra/query/scaladsl/CassandraReadJournal.scala

```diff
@@ -468,6 +550,35 @@ class CassandraReadJournal(system: ExtendedActorSystem, config: Config)
       EventEnvelope(offset, persistentRepr.persistenceId, persistentRepr.sequenceNr, payload)
     }
 
+  private[this] def toEventEnvelope2(persistentRepr: PersistentRepr, offset: Offset): immutable.Iterable[EventEnvelope2] =
+    adaptFromJournal(persistentRepr).map { payload =>
+      EventEnvelope2(offset, persistentRepr.persistenceId, persistentRepr.sequenceNr, payload)
+    }
+
+  private[this] def eventEnvelope2toUuidEnvelope(env: EventEnvelope2): UUIDEventEnvelope =
+    UUIDEventEnvelope(
+      env.offset match {
+        case TimeBasedUUID(uuid) => uuid
+        case unexpected          => throw new IllegalArgumentException(s"Unexpected offset type $unexpected for envelope $env")
+      },
+      env.persistenceId,
+      env.sequenceNr,
+      env.event
+    )
+
+  private[this] def internalUuidToOffset(uuid: UUID): Offset =
+    if (uuid == firstOffset) NoOffset
+    else TimeBasedUUID(uuid)
+
+  private[this] def offsetToInternalOffset(offset: Offset): UUID =
+    offset match {
+      case TimeBasedUUID(uuid) => uuid
+      case Sequence(timestamp) => offsetUuid(timestamp)
```

Comment:
You are correct, I think this was a leftover.

The `timeBasedUUIDFrom` as a public method?